.github/workflows/conformance-ingress.yaml

```diff
@@ -111,7 +111,8 @@ jobs:
             --set operator.image.pullPolicy=IfNotPresent \
             --set operator.image.useDigest=false \
             --set securityContext.privileged=true \
-            --set ingressController.enabled=true
+            --set ingressController.enabled=true \
+            --set debug.enabled=true
```

Comment:
this is good idea, I have incorporated the same in https://github.com/cilium/cilium/pull/19825

Reply:
If you're going to carry half of this patch in your PR, would you mind carrying the other half as well (i.e., changes to `conformance-kind.yaml`)? :grin:

Reply:
Sure 💯

Reply:
Done :heavy_check_mark: